src/web/main.js

```diff
@@ -0,0 +1,80 @@
+window.onload = function(){
+
+var interval, cnt, startButton, stopButton
+
+cnt = 0;
+
+// Create graph lines	
+Plotly.plot("chart",[{
+  y:[],
+  type:'line'
+}], {
+  yaxis: {
+    range: [300, 1015]
+  }
+});
+
+startButton = document.getElementById("startButton")
+stopButton = document.getElementById("stopButton")
+
+startButton.onclick = function() {
+  alert("Make sure the sensor is in the soil")
+  startButton.style.display = "none"
+  stopButton.style.display = "block"
+ 
+  interval = setInterval(async function(){
+    reading = await getReading();
+    eel.create(reading)();
+    plotGraph(reading);  
+    console.log(reading);     
+  }, 1000);
+
+stopButton.onclick = function() {
+  startButton.style.display = "block"
+  stopButton.style.display = "none"
+  clearInterval(interval)
+}
+
+
+
+}
+
+function plotGraph(reading){
+  Plotly.extendTraces("chart",{y:[[reading]]},[0]);
+cnt++;
+
+//moving y axis along 
+  if(cnt > 20) {
```

Comment:
Could we alter this to show the time when the reading was taken?

Reply:
Potentially! Is it high priority?

Reply:
Not really, but something to consider